docs/issues_catched.md

```diff
@@ -1,28 +1,23 @@
 
 # Issues Precaution finds
 
-Precaution utilizes multiple security static code analysis tools.
-That is the reason why the amount and types of checks for the different
-programming languages differs.
-The security static code analyzers are language-focused
-and some of the vulnerabilities they find are specific 
-for the programming language.
+Precaution utilizes multiple security static code analysis tools (linters)
+which are language-focused identify issues which may be specific to the language they target
```

Comment:
this line isn't quite right:
"which are language-focused _and therefore_ identify issues which may be specific to the language they target"?

Reply:
Do you think that now is better?

Reply:
LGTM